blackbox/docs/admin/cluster-management.rst

```diff
@@ -0,0 +1,101 @@
+.. _cluster_management:
+
+==================
+Cluster management
+==================
+
+With the ``crate-node`` CLI tool you can manage your CrateDB clusters by forming
+new clusters, repairing broken clusters, and by repairing broken nodes.
+
+The ``crate-node`` CLI tool
+---------------------------
+
+The ``crate-node`` CLI tool lets you:
```

Comment:
explain where the tool is found and how to run it for the first time. like https://crate.io/docs/crate/reference/en/latest/run.html

perhaps the first run instruction should be the `--help` option